views/js/runtime/qtiCustomInteractionContext.js

```diff
@@ -75,31 +75,14 @@ define(function(){
          * @param {string} pciTypeIdentifier
          * @returns {Object} clonedPciModel
          */
-        createPciInstance : function(pciTypeIdentifier){
+        createPciInstance: function(pciTypeIdentifier) {
```

Comment:
You should use object shorthand here. It will fix the code style issue with the anonymous function and will be compliant with ES6 code style as well.